opm/common/OpmLog/LogBackend.cpp

```diff
@@ -27,18 +27,37 @@ namespace Opm {
     {
     }
 
+    LogBackend::~LogBackend()
+    {
+    }
 
-    bool LogBackend::includeMessage(int64_t messageFlag) {
-        if (((messageFlag & m_mask) == messageFlag) &&
-            (messageFlag > 0))
-            return true;
-        else
-            return false;
+    void LogBackend::configureDecoration(bool use_prefix, bool use_color_coding)
```

Comment:
I like the idea, but I think it is a little too much for the poor LogBackend, when we need this kind of detail we should perhaps make a separate MessageDecorator or MessageFormatter class?